.github/workflows/integration-test-upgrade-suite.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
         run: |
           tests/scripts/github-action-helper.sh collect_udev_logs_in_background
           export DEVICE_FILTER=$(lsblk|awk '/14G/ {print $1}'| head -1)
-          go test -v -timeout 1800s -failfast -run CephUpgradeSuite/TestUpgradeRook github.com/rook/rook/tests/integration
+          go test -v -timeout 2400s -failfast -run CephUpgradeSuite/TestUpgradeRook github.com/rook/rook/tests/integration
```

Comment:
We also need this change in integration-tests-on-release.yaml for the master/release CI.

Reply:
fixed #12656